.claude/hooks/deployment-monitor.sh

```
@@ -0,0 +1,140 @@
+check_process() {
+    local pid=$1
+    if ps -p "$pid" > /dev/null 2>&1; then
+        return 0  # Process is running
+    else
+        return 1  # Process is not running
+    fi
+}
+
+# Function to monitor deployment
+monitor_deployment() {
+    local start_time=$(date +%s)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)